.github/workflows/build-publish-documentation-website.yml

```diff
@@ -2,6 +2,9 @@ on:
   push:
     branches:
       - master
+  pull_request:
+    paths:
+      - 'docs/*'
```

Comment:
[It looks like we want two stars (`**`), not one.](https://help.github.com/en/articles/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet)

Reply:
Thanks, fixed now!